__tests__/app.test.js

```diff
@@ -263,4 +263,27 @@ describe("DELETE /api/comments/:comment_id", ()=>{
   test("400: should respond with a 400 status on invalid comment_Id", () => {
     return request(app).delete("/api/comments/fakeId").expect(400);
   });
-})
\ No newline at end of file
+})
+
+describe("GET /api/users", () => {
+  test("Responds with an array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const {
+          body: { users },
+        } = response;
+        users.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
```

Comment:
It would be slightly more secure to use `typeof` assertions here instead, so that you can check not only the properties exist but also that they hold the expected data types 💡